src/runtime/offers-flow.js

```diff
@@ -225,19 +231,22 @@ export class OffersFlow {
     return Promise.resolve();
   }
 
+  shouldShow_(clientConfig) {
```

Comment:
is it worth naming this method "shouldShowAutoPrompt" for additional clarity?

Reply:
also, please add a JSDoc comment so there's type information for the argument.

Reply:
Actually intentionally keeping it more generic (just tied to showing of offers in this context)  as just re-using the UI predicates which both return the same thing but eventually may be tweaked and expanded.

Reply:
sounds good!